build_debian.sh

```diff
@@ -228,6 +228,7 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     python-scapy            \
     tcptraceroute           \
     mtr-tiny                \
+    ipmitool                \
```

Comment:
ipmitool is installed the pmon container, can you use that?

Reply:
Done.